src/researchhub_document/related_models/feed_exclusion_model.py

```diff
@@ -0,0 +1,24 @@
+from django.db import models
+from typing_extensions import Required
+
+from researchhub_document.related_models.researchhub_unified_document_model import (
+    ResearchhubUnifiedDocument,
+)
+from utils.models import DefaultModel
+
+
+class FeedExclusion(DefaultModel):
+    # Not a foreign key because hub_id=0 is homepage
+    hub_id = models.IntegerField(
+        default=False,
+        db_index=True,
+    )
+
+    unified_document = models.ForeignKey(
+        ResearchhubUnifiedDocument,
+        default=False,
+        blank=False,
+        null=False,
```

Comment:
These two should be True?

Reply:
I think they should be false. I never want this value to be null or blank since I'm excluding unified documents from a particular feed.

Reply:
Oh yeah you're correct. I mixed it up in my head.